easybuild/easyconfigs/c/CCL/CCL-1.12.1-GCCcore-10.3.0.eb

```
@@ -0,0 +1,51 @@
+easyblock = 'MakeCp'
+name = 'CCL'
+version = '1.12.1'
+
+homepage = 'https://ccl.clozure.com/'
+description = """Clozure CL (often called CCL for short) is a free Common Lisp
+ implementation with a long history. Some distinguishing features of the implementation
+ include fast compilation speed, native threads, a precise, generational, compacting
+ garbage collector, and a convenient foreign-function interface."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+source_urls = ['https://github.com/Clozure/ccl/releases/download/v%(version)s']
+sources = ['%(namelower)s-%(version)s-linuxx86.tar.gz']
+checksums = ['923f2979144e96306cf5124800444e1c5cc0295fa9babb3ffc47a5631344b0aa']
+
+builddependencies = [
+    ('binutils', '2.36.1'),
+]
```

Could you also add M4 as build dependency (see https://github.com/easybuilders/easybuild-easyconfigs/pull/15075)?
```suggestion
    ('M4', '1.4.18'),
]
```

---

sure, thanks for the heads up on that issue